src/containers/Vendor/Profile/index.js

```diff
@@ -1,108 +0,0 @@
-import React, { Component } from 'react'
```

Comment:
This profile stuff was useful actually. Might need to be resurrected in the "my reputation" view. Along the lines of aggregate scores etc you were talking about. I suppose we can do new components for it. It is nothing too complicated.